apps/api/app.py

```
@@ -0,0 +1,128 @@
+    logger.info("Starting Taboot API", extra={"version": "0.4.0"})
+
+    try:
+        redis_client = await redis.from_url(
+            config.redis_url,
+            encoding="utf-8",
+            decode_responses=True,
+        )
+        app.state.redis = redis_client
+        logger.info("Redis client initialized", extra={"url": config.redis_url})
+    except Exception as e:
+        logger.error("Failed to initialize Redis client", extra={"error": str(e)}, exc_info=True)
```

Ruff (0.14.1)

Logging `.exception(...)` should be used instead of `.error(..., exc_info=True)`

(G201)